workspace/__lib__/xod-dev/ds-rtc/project.xod

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "Support for the DS1302/DS1307/DS3231 based breakout RTC boards"
```

Comment:
It’s not a name, but the description. Version, license is missing. Use other’s library `project.xod` as a template.